frontend/app/components/Spots/SpotPlayer/components/AccessModal.tsx

```diff
@@ -107,13 +102,18 @@ function AccessModal() {
     setTimeout(() => setIsCopied(false), 2000);
   };
 
+  const formatExpirationTime = (seconds: number) => {
```

Comment:
we already have formatting functions in utils

Reply:
As I see, the function in utils limited to minutes and seconds, where as the one in this file addresses more flexible format covering hours, days and weeks. So the function in utils isn't solving exactly what formatExpirationTime does.

Reply:
Well its better to add something to what we have already instead of reinventing the wheel

Reply:
For example adding "format" prop with default value that will equal to what it had before